Configuration/Routes/Example.yml

```diff
@@ -0,0 +1,57 @@
+routeEnhancers:
+  Products:
+    type: CombinedFacetEnhancer
+    limitToPages:
+      - 10
+    extensionKey: tx_solr
```

Comment:
People often start copying over Examples from docs, how about we leave out the things people don't need to set all the time? What do you think? "extensionKey" for instance.

In general we could give the type of the enhancer a more "speaking" name. a simple "SolrFacets" and we could leave out the "solr:" sublevel then? Just a suggestion, not a requirement.

Reply:
The parameter `extensionKey` could be omitted. Internally it is set to `tx_solr` if not configured.

The idea behind use `solr` as sublevel ist to avoid side effects in case we using key words that may be reserved for TYPO3.

In case `solr` should be dropped, we need an identifier to avoid side effects like:

```yaml
facet-path:
      multiValueSeparator: '+'
      replaceCharacters:
        ' ': '_'
facet-query:
      multiValueSeparator: '+'
      replaceCharacters:
        ' ': '_'
      concat: true
      # valueSeparator: ','

     mask: true
     map:
       taste: taste
```

Reply:
The name of the enhancer changes into `SolrCombinedFacetEnhancer` since the task of this enhancer is to combine and mask the facets.